.github/workflows/codeql.yml

```diff
@@ -37,6 +37,9 @@ jobs:
         uses: github/codeql-action/init@v2
         with:
           languages: ${{ matrix.language }}
+          config: |
+            paths:
+              - src
```

Comment:
probably makes sense to remove lines `:10` and `:15` as well, to not trigger this action redundantly

Reply:
I think we still need these so CodeQL is triggered when `dist` has changed, which means a dependency in `package.json` has been updated and therefore might change the behavior of the project.

Reply:
:ack ok that makes sense to me. Thanks!